libs/clients/driving-license/src/lib/utils.ts

```diff
@@ -0,0 +1,28 @@
+import fetch from 'isomorphic-fetch'
+import { logger } from '@island.is/logging'
+
+// TODO: Copy paste from VMST client - a really helpful helper
+
+export const createWrappedFetchWithLogging = (
+  input: RequestInfo,
+  init?: RequestInit,
+): Promise<Response> => {
+  return new Promise((resolve, reject) => {
+    fetch(input, init)
```

Comment:
Isn't it strange to wrap `fetch` in a Promise since it is already returning a pretty useful promise?

Reply:
yes, this is a copy paste from vmst client, without much thought.. @jeremybarbet already pointed me to a shared utils thing that is basically the same thing but better